lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss

```diff
@@ -162,7 +162,7 @@
 
 .slideshow__controller {
   display: table;
-  margin: 0 auto;
+  margin: $base-spacing auto;
```

Comment:
Ok, nevermind. Change the `top` back to `0`. Don't use a percentage for this value. And then just use margins on the `.slideshow__controller` like you had before _but_ only add a top margin:

```
margin: $base-spacing auto 0;
```